### PR TITLE
#1: Added example dashboard

### LINKED ISSUE #1: Add example dashboard
Add an example dashboard for the metrics reported by our cloud-watch adapter.

### LINKED ISSUE #3: Add alarm
Add an alarm when TEMP_DB_RAM > 70% * RAM

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #1
Closes #3

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Exasol
```
Comment (morazow) [inline LICENSE]:
```suggestion
Copyright (c) 2021 Exasol
```

Comment (morazow) [reply LICENSE]: This should also be changed in the keeper.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Exasol CloudWatch Example Dashboard
+
+This project contains an [AWS CDK](https://aws.amazon.com/cdk/) template for an example CouldWatch dashboard that visualizes the metrics reported by the [Exasol – Cloud Watch adapter](https://github.com/exasol/cloudwatch-adapter).
+
+This dashboard should fasten your start with Exasol and CloudWatch.
```
Comment (morazow) [inline README.md]:
```suggestion
This dashboard should get you started with Exasol and CloudWatch integration.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+There you will have to set a Deployment Name. This is the name of your exasol installation you entered during the setup of the cloudwatch-adapter.
+
+## Advanced Usage
+
+In this repository we define the dashboard by code using [AWS CDK](https://aws.amazon.com/cdk/). That allows us to define the dashboard in a very readable way.
+
+You can use this as a starting point for your own dashboards. For that you need to go through the following steps:
+
+* Checkout (or fork) this repository
+* Adopt the Dashboard to your needs (modify `src/main/java/com/exasol/cloudwatchexampledashboard/CloudwatchDashboardExamplesStack.java`)
```
Comment (morazow) [inline README.md]:
```suggestion
* Adapt the dashboard to your needs (modify `src/main/java/com/exasol/cloudwatchexampledashboard/CloudwatchDashboardExamplesStack.java`)
```

Comment (morazow) [reply README.md]: Would this be possible without modifying the java code? For example, using the a config file (yaml, props, etc).

**Review by morazow — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (jakobbraun) [reply LICENSE]: yes. I created an issue for it: https://github.com/exasol/project-keeper-maven-plugin/issues/69

**Review by morazow — APPROVED** [review]

**Pull request merged**